app/assets/javascripts/components/activity/activity_table.cjsx

```diff
@@ -39,9 +43,9 @@ ActivityTable = React.createClass(
 
     drawers = @state.activity.map (revision) ->
       courses = revision.courses.map (course) ->
-        <li><a href="/courses/#{course.slug}">{course.title}</a></li>
+        <li key={Math.random() * course.id}><a href="/courses/#{course.slug}">{course.title}</a></li>
 
-      <tr className='activity-table-drawer'>
+      <tr key={Math.random() * revision.key} className='activity-table-drawer'>
```

Comment:
Fixed via 841da57. With a higher article limit, I saw revisions coming through where the id and course id were duplicates, though that doesn't seem possible. We'll have to keep an eye on that.